variables.tf

```
@@ -28,7 +28,7 @@ variable "astronomer_chart_git_repository" {
 }
 
 variable "astronomer_version" {
-  description = "Verison of Helm chart to use, do not include a 'v' at the front"
+  description = "Version of Helm chart to use, do not include a 'v' at the front"
   default     = "0.12.0-alpha.1"
```

We should definitely bump this up to at least a supported version.
```suggestion
  default     = "0.29.2"
```